Maze/MazeController.swift

```diff
@@ -67,8 +67,9 @@ class MazeController {
     }
     
     // MARK: Adding Maze Objects
-    
-    func addMazeObject(var object: MazeObject) {
+    func addMazeObject(obj: MazeObject) {
+        
+        var object = obj
```

Comment:
Not a fan of rewriting it like this; should we consider refactoring the way the Maze works? Shouldn't the MazeView be responsible for adding a maze object (similar to how UIViewController/UIView work)?